aten/src/ATen/Config.h.in

```diff
@@ -10,3 +10,9 @@
 #define AT_MKL_ENABLED() @AT_MKL_ENABLED@
 #define AT_NNPACK_ENABLED() @AT_NNPACK_ENABLED@
 #define CAFFE2_STATIC_LINK_CUDA() @CAFFE2_STATIC_LINK_CUDA_INT@
+#define AT_BUILD_WITH_BLAS() @USE_BLAS@
```

Comment:
what does the @ do? where do you undef this macro?

Reply:
`@SOMETHING@` is a cmake variable and will be replaced by cmake with its value:
https://cmake.org/cmake/help/v3.2/command/configure_file.html
> Copies an <input> file to an <output> file and substitutes variable values referenced as `@VAR@` or `${VAR}` in the input file content. Each variable reference will be replaced with the current value of the variable, or the empty string if the variable is not defined. Furthermore, input lines of the form:

Reply:
Do I need to undef it? I didn't see other macros in this file being undefed.

Reply:
The internal build needs to get adjusted at https://our.internmc.facebook.com/intern/diffusion/FBS/browse/master/fbcode/caffe2/aten/TARGETS?commit=74d1f672dfef08d13ba2f2b33ca93d36ecc796df&lines=456